explainer.md

```diff
@@ -613,6 +627,33 @@ This will make it possible to:
     for example, detecting the exposed role of an element
     and modifying the accessible help text to suit.
 
+#### Why is accessing the computed properties being addressed last?
+
+**Consistency**
+Currently, the accessibility tree is not standardized between browsers:
+Each implements accessibility tree computation slightly differently.
+In order for this API to be useful,
+it needs to work consistently across browsers,
+so that developers don't need to write special case code for each.
+
+Before we can create an API which allows accessing the computed properties,
```

Comment:
This sentence is kind of redundant with the previous paragraph. How about something like, we believe it will be possible to create an API that's consistent but it's significantly more complicated than the first three phases.